### PR TITLE
tech: Update Price component to Vanilla-Extract

### PR DESCRIPTION
**Pull request by Youakeem**
## Describe your changes
Subj

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by Youakeem** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4699?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4700** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4700?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4699** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4699?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4698** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4698?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4697** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4697?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4696** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4696?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4690** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4690?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Youakeem and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by alebedev — APPROVED** [review]

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/components/Price.tsx [hunk 1]
```diff
@@ -16,31 +16,30 @@ type Props = {
...
+  currencyCode,
+  className,
   ...props
 }: Props) => {
   const formatter = useFormatter()
 
   return (
-    <Wrapper>
-      {props.reducedAmount !== undefined && (
+    <div className={clsx(xStack({ gap: 'xs' }), className)} {...props}>
```
Comment (gustaveen) [inline apps/store/src/components/Price.tsx:28]: Don't remember if I asked this in on of the other PRs :D But do we need to pass a className prop? For most components we don't extend the usage right?

**Review by gustaveen — APPROVED** [review]
👍

File: apps/store/src/components/Price.tsx (hunk 1 above)
Comment (Youakeem) [reply apps/store/src/components/Price.tsx:28]: I think it's good to be able to pass classnames, we are spreading the rest of props anyways. When composing different components it's common to apply extra classes/styles.

**Comment by Youakeem** [comment]
### Merge activity

* **Sep 19, 3:16 AM EDT**: @Youakeem started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4699).

**Pull request merged**